### DISCUSSION TITLE
Welcome to devika Discussions!

### DISCUSSION THREAD

**Discussion by mufeedvh** [stitionai/devika#289] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ExistedAura7234** [comment]
Hola soy ExistedAura7234 y soy un joven programador

**Reply by entrepeneur4lyf** [reply]
Bienvenido, Señor Joven Programador!

**Comment by ExistedAura7234** [comment]
Tengo una pregunta: ¿Comó m****** se hace para iniciar el devika? Las instrucciones NO sirven :( Pongan un tuto de yt o no va a ir a ningun lado esta IA

**Reply by entrepeneur4lyf** [reply]
Después de intentarlo muchas veces, pude hacerlo funcionar en Ubuntu. Sin embargo, la interfaz se congela y el agente se detiene después de uno o dos intentos. Mi recomendación es intentar utilizar OpenDevin. Pude hacer que eso funcionara.

**Reply by ExistedAura7234** [reply]
Te felicito ídolo, pero mi problema no es que se congele, es que no tengo la más p*** idea de como se inicia la app porque las instrucciones pareciera que las hubiera escrito un politico de mi pais >:( En serio, no se si el desarrollador no se da cuenta, pero la mitad de los usuarios de git hub escribimos en C#, NO EN Phyton! Así que por favor explicame, como hiciste para que te workeara el f****** Devinka, preferiblemente de manera detallada. Gracias!

**Comment by shrawank039** [comment]
What will be the future of this browser-based approach to fixing code? If these LLMs can fine-tune themselves just by reading any article once. Then no one is going to write that much resource on the internet.

**Comment by lesego94** [comment]
I am very new to Devika, could I use it on an existing local repo to modify existing code and run tests?

**Reply by nickfixit** [reply]
Yes
